x.py

```diff
@@ -22,7 +22,7 @@
             pass
 
 rust_dir = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(rust_dir, "src", "bootstrap"))
+sys.path[:0] = [os.path.join(rust_dir, "src", "bootstrap")]
```

Comment:
Why this change?

Reply:
Might make it a bit faster because it doesn't have to check all the other functions. Also theoretically someone could have something with the name `bootstrap` installed

Reply:
So it is like a `prepend`, so that it can still find the other deps? (sorry, not python master)

Reply:
Yes, I'll add a short comment

Reply:
Is doing it this way better than through `sys.path.insert(0, os.path.join(rust_dir, "src", "bootstrap"))`?
It seems clearer when you don't have a Python Indexing PhD

Reply:
I guess it isn't

Reply:
You could have left your comment, it is always good to know why this path manipulation happens.